openhands/agenthub/codeact_agent/tools/str_replace_editor.py

```diff
@@ -4,10 +4,12 @@
 
 _DETAILED_STR_REPLACE_EDITOR_DESCRIPTION = """Custom editing tool for viewing, creating and editing files in plain-text format
 * State is persistent across command calls and discussions with the user
-* If `path` is a file, `view` displays the result of applying `cat -n`. If `path` is a directory, `view` lists non-hidden files and directories up to 2 levels deep
+* If `path` is a file, `view` displays the file content in Markdown format. If `path` is a directory, `view` lists non-hidden files and directories up to 2 levels deep
```

Comment:
```suggestion
* If `path` is a text file, `view` displays the file content in `cat -n`. If `path` is a directory, `view` lists non-hidden files and directories up to 2 levels deep
```

@ryanhoangt is this accurate? We only show other file extensions as markdown, right?